docusaurus/docs/reactnative/guides/push_notifications_v1.mdx

```diff
@@ -1,14 +1,16 @@
 ---
-id: push-notifications
+id: push-notifications-v1
 sidebar_position: 1
```

Comment:
does it require changing?

Reply:
the `id` you mean? Or label

Reply:
`sidebar_position`

Reply:
right now, index is same it fallbacks to label I guess